lib/models/application.coffee

```diff
@@ -376,34 +376,51 @@ getApplicationModel = (deps, opts) ->
 	# @function
 	# @memberof resin.models.application
 	#
-	# @param {String} name - application name
-	# @param {String} deviceType - device type slug
-	# @param {(Number|String)} [parentNameOrId] - parent application name or id
+	# @param {Object} options - application creation parameters
+	# @param {String} options.name - application name
+	# @param {(Number|String)} options.applicationType - application type slug or id
```

Comment:
It'd be nice to include an example slug in here, since people might not know what this is (although I guess in future they will... Useful anyway). `e.g. microservice-starter`

Reply:
Just noticed - this is actually optional in the current API implementation, it'd be nice to have it optional here too.

Reply:
Didn't know how the API would end up, so I decided to be stricter for start and strictly fit our UI use case. It seems that the API isn't going to change. Will change this :+1:

Reply:
Isn't the examples section good enough to demonstrate the available slugs?
Regardless, I'm adding an example in the `@param` as well.